TestAssets/AsIsProjects/XprojCsprojProjects/BasicProject_invalid-missing-dll/Sample1-Xproj+Csproj/artifacts/src/ConsoleApp13/obj/Debug/net46/dotnet-compile.rsp

```diff
@@ -0,0 +1,20 @@
+--temp-output:C:\Users\micodoba\OneDrive for Business\tests\TestXprojCsproj-master\Sample1-Xproj+Csproj\artifacts\src\ConsoleApp13\obj\Debug\net46\
```

Comment:
If possible, we should try to find a better pattern here. 

Checking in all of NewtonSoft.Json :frowning: